src/admin/modules/pitch/components/PitchesList.tsx

```diff
@@ -64,20 +64,20 @@ const ApplicationDataGrid = () => {
     <Datagrid rowClick={"show"}>
       <TextField source="project_name" label="Project Name" sortable />
       <ReferenceField
-        source="organisation_id"
+        source="organisationId"
         label="Organization"
         reference={modules.organisation.ResourceName}
         link="show"
       >
         <TextField source="name" />
       </ReferenceField>
       <SimpleChipFieldArray
-        source="restoration_intervention_types"
+        source="restorationInterventionTypes"
         label="Restoration Intervention Types"
         choices={optionToChoices(getRestorationInterventionTypeOptions())}
       />
-      <SelectField source="project_country" label="Countries" choices={countryChoices} />
-      <DateField source="created_at" label="Date Added" locales="en-GB" />
+      <SelectField source="projectCountry" label="Countries" choices={countryChoices} />
+      <DateField source="createdAt" label="Date Added" locales="en-GB" />
```

Comment:
Please double check that these fields are all included. For instances, `createdAt` does not appear to be on the v3 DTO, unless I'm missing something.